rest-api/dao/participant_dao.py

```diff
@@ -194,14 +266,20 @@ def to_client_json(self, model):
   def from_client_json(self, resource_json, id_=None, expected_version=None, client_id=None):
     parse_json_enum(resource_json, 'withdrawalStatus', WithdrawalStatus)
     parse_json_enum(resource_json, 'suspensionStatus', SuspensionStatus)
+    get_site_id_from_google_group(self, resource_json)
+    get_awardee_id_from_name(self, resource_json)
+    get_organization_id_from_external_id(self, resource_json)
     # biobankId, lastModified, signUpTime are set by DAO.
     return Participant(
         participantId=id_,
         version=expected_version,
         providerLink=json.dumps(resource_json.get('providerLink')),
         clientId=client_id,
         withdrawalStatus=resource_json.get('withdrawalStatus'),
-        suspensionStatus=resource_json.get('suspensionStatus'))
+        suspensionStatus=resource_json.get('suspensionStatus'),
+        organizationId=resource_json.get('organizationId'),
```

Comment:
As noted above: you could instead do:

organizationId=get_organization_id(resource_json, self.organization_dao),
siteId=get_site_id(resource_json, self.site_dao),

etc...

Also see above: the API fields could be 'awardee', 'organization', and 'site'... we don't need to accept 'hpoId' for backwards compatibility purposes, since previously the only way to modify pairing was using providerLink.